### PR TITLE
kotlin bindings

### PR DESCRIPTION
**Pull request by ganfra**
It's time to re-open this PR.
There are 2 scripts to generate .aar files :
- one for matrix-sdk-crypto-ffi crate
- one for matrix-sdk-ffi create

### CODE REVIEW

File: bindings/kotlin/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Matrix rust components kotlin
+
+This project and build scripts demonstrate how to create an aar and how to import it in your android projects.
+
+## Prerequisites
+
+* the Rust toolchain
+* UniFFI - `cargo install uniffi_bindgen`
```
Comment (gnunicorn) [inline bindings/kotlin/README.md:8]: we have recently deprecated the sh from apple and moved all into an xtask job - an important reason being that we don't have uniffi_bindgen mismatches anymore. Could we maybe do the same here? I'd rather not add bash-scripts again :( ...

**Review by gnunicorn — CHANGES_REQUESTED** [review]
Thanks for this!

Could we please:
 - move the bash scripts into `xtask` tasks?
 - add a github workflow to ensure this actually works?

File: bindings/kotlin/README.md (hunk 1 above)
Comment (ganfra) [reply bindings/kotlin/README.md:8]: Can we do that later?

**Review by gnunicorn — APPROVED** [review]
okay, moved the missing pieces into an issue so we can merge this for now.

**Pull request merged**